src/config.ts

```diff
@@ -70,9 +75,16 @@ const configSchema = {
         remotePackage: {type: 'array', items: {type: 'string'}},
         remoteUrl: {type: 'string'},
         sourceMap: {oneOf: [{type: 'boolean'}, {type: 'string'}]},
+        stat: {type: 'boolean'},
```

Comment:
This isn’t part of this PR, but I noticed `stat` was missing from the validation here (I think because they were merged around the same time). Happy to break this out into its own PR if needed though.